Mathlib/MeasureTheory/Measure/Lebesgue/Basic.lean

```diff
@@ -577,6 +577,95 @@ theorem volume_regionBetween_eq_lintegral [SigmaFinite μ] (hf : AEMeasurable f
           (regionBetween_subset (AEMeasurable.mk f hf) (AEMeasurable.mk g hg) s)).symm
 #align volume_region_between_eq_lintegral volume_regionBetween_eq_lintegral
 
+-- Place where?
```

Comment:
close to `ae_prod_mem_iff_ae_ae_mem` in `MeasureTheory.Constructions.Prod.Basic`? Would any of the lemmas around help here, by the way?

Reply:
Oh, this lemma is newer than my branch. I will merge master and see. Thanks!

And thank you for the suggested placement! I intended to think about it once I got cache from the very low level file modification. Incidentally, I again had trouble getting `#find_home` to report a place --- maybe I'm using it incorrectly. But your suggestion looks like a perfect place now.

Reply:
I did not see a way to use the earlier results. The main reason, perhaps, is that most of the earlier results assume measurability of the set in the product space and these new lemmas are about null-measurability. My proofs of the trivialities `measure_prod_compl_eq_zero` and `NullMeasurableSet.prod` do seem suspiciously lengthy, but I don't find golfs now.